static/styles/zulip.css

```diff
@@ -678,8 +678,11 @@ li.actual-dropdown-menu i {
     margin-right: 3px;
 }
 
-.settings-dropdown-cog {
-    padding: 6px 12px;
+.settings-dropdown-avatar {
+    margin-top: -6px;
+    padding: 10px;
+    border-radius: 12px;
+    width: 25px;
 }
```

Comment:
The fading/hover behavior for this seems weird.  I also worry a bit that the widget is a bit too invisible when it's a gravatar (since it looks like basically a random pattern), which will be the case for most new users.  

Can you research what other tools using an avatar in this slot do for users without an avatar?  We may need to do some other decoration (Whether the word "settings", an icon, or something else) to make it clear.